.github/workflows/images.yaml

```diff
@@ -75,6 +75,8 @@ jobs:
         env:
           DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
           DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
+          QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD  }}
+          QUAY_USERNAME: ${{ secrets.QUAY_USERNAME  }}
         with:
           entrypoint: make
           args: -C images cilium-test-image PUSH=true
```

Comment:
You can just add `REGISTRIES="docker.io/cilium quay.io/cilium"` on this line to achieve what you want the title of this PR suggests.